src/components/Requirements/IncompleteSelfCheck.vue

```diff
@@ -1,13 +1,11 @@
 <template>
   <div class="incompleteselfcheck">
     <new-self-check-course-modal
-      class="incompleteselfcheck-modal"
       v-model="isCourseModalOpen"
       v-if="isCourseModalOpen"
```

Comment:
Should've mentioned this on the older PR, but this form of opening a modal kind of counteracts the whole point of v-model. I get that the v-if will reset the data inside the modal, but maybe we need to look towards an older way of implementing the body of the modal. That is, make the body of the modal as a component and instead use TeleportModal where the current modal components are defined right now. Or, you could instead have two components per modal (a little much, which is why I sort of dislike having Single File Components) which forward props to the modal body component.  The two-component option might be better if the modal has functionality that requires the modal body component to need to listen to events from modal buttons for purposes other than closing an opening. If we end up going with the v-if solution, the v-model serves technically no purpose and the modal may as well be always open. This is quick so you can do it in this PR if you want, but you can also add it in the todo list.